File: PHP/LeetCode/Count Pairs Whose Sum is Less than Target.php
// https://leetcode.com/problems/count-pairs-whose-sum-is-less-than-target/

class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer
     */
    function countPairs($nums, $target) {
        $pairCount = 0;

        for ($i = 0; $i < count($nums)-1; ++$i) {
            for ($j = $i+1; $j < count($nums); ++$j) {
                if ($nums[$i] + $nums[$j] < $target) {
                    $pairCount++;
                }
            }
        }
        
        return $pairCount;
    }

}

File: Java/LeetCode/Count Pairs Whose Sum is Less than Target.java
// https://leetcode.com/problems/count-pairs-whose-sum-is-less-than-target/

class Solution {

    public int countPairs(List<Integer> nums, int target) {
        int pairCount = 0;

        for (int i = 0; i < nums.size(); ++i) {
            for (int j = i+1; j < nums.size(); ++j) {
                if (nums.get(i) + nums.get(j) < target) {
                    pairCount++;
                }
            }
        }

        return pairCount;
    }

}
